slack.mdx

```diff
@@ -28,6 +28,15 @@ Once you’ve set up the integration in Plain, go to the Slack channels you want
   src="https://static-assets.plain.com/docs/slack/add-to-slack.mp4"
 ></video>
 
+<Warning>
+  When Plain is added to a channel, we automatically classify it as a Customer channel if it's a slack connect channel or one for [internal discussions](/thread-discussions) otherwise.
```

Comment:
```suggestion
  When Plain is added to a channel, we automatically classify it as a Customer channel if it's a _Slack Connect_ channel or one for [internal discussions](/thread-discussions) otherwise.
```